spec/features/search_spec.rb

```diff
@@ -115,4 +115,9 @@
     expect(page).to have_content "Welcome!"
     expect(page).not_to have_selector "#q[value='history']"
   end
+
+  it "sanitizes searches with unconfigured facet parameters" do
+    visit root_path f: { missing_s: [1] }
+    expect(page).not_to have_text "Missing S"
```

Comment:
I think this is a useful thing to check, but I'm wary of "page doesn't have text" tests - high likelihood of false passes. I wonder if we could now re-implement this as a ConstraintsComponent test?

Reply:
Sounds good.